content/300-guides/100-performance-and-optimization/100-prisma-client-transactions-guide.mdx

```diff
@@ -181,17 +181,6 @@ const createTeamOperation = prisma.team.create({
 await prisma.$transaction([createUserOperation, createTeamOperation])
 ```
 
-##### Are cascading deletes supported by the Prisma Client nested writes?
-
-<Admonition type="warning">
-
-In [2.26.0](https://github.com/prisma/prisma/releases/tag/2.26.0) and later it is possible to do cascading deletes using [referential actions](/concepts/components/prisma-schema/relations/referential-actions).
-
-</Admonition>
-
-- Use the `$transaction` API to perform a cascading delete. Deleting a team and deleting the team's members are not dependent writes - if you know the ID of the team, you can [perform both operations within a `$transaction`](#scenario-privacy-legislation).
-- [Configure cascading deletes at the database level](/guides/database/advanced-database-tasks/cascading-deletes).
```

Comment:
Can this section just be deleted or do we need a version of it around still?

Reply:
Investigation: https://prisma-company.slack.com/archives/CC59J6VEY/p1667508837505919

Reply:
Could not get definitive information, but probably not that relevant any more. People should upgrade.